client/packages/system/src/Item/context/useUnitVariant.ts

```diff
@@ -46,33 +55,64 @@ export const useInitUnitStore = () => {
   // TODO add user selected from app data
 };
 
-export const useUnitVariant = (itemId: string) => {
-  // TODO [state.items[itemId], state.userSelected[itemId]]
-  const item = useUnitStore(state => state.items[itemId], isEqual);
-  if (!item) {
+export const useUnitVariant = (
```

Comment:
this diff is a bit weird, i added type return and then extra selector for useUnitStore